txpool/tx_pool_test.go

```diff
@@ -520,18 +520,9 @@ func TestExpiredTxs(t *testing.T) {
 		assert.NoError(t, pool.Add(newTx(pool.repo.ChainTag(), nil, 21000, tx.BlockRef{}, 100, nil, tx.Features(0), devAccounts[i%len(devAccounts)])))
 	}
 
-	var wg sync.WaitGroup
-	wg.Add(1)
-
-	go func() {
-		defer wg.Done()
-		pool.housekeeping()
-	}()
-
-	time.Sleep(1 * time.Second)
-	pool.cancel()
+	executables, _, _ := pool.wash(pool.repo.BestBlockSummary())
+	pool.executables.Store(executables)
```

Comment:
lgtm, I did notice that the other tests are also using a time.sleep ? Do you think it's worth updating them ?